docs/conf.py

```diff
@@ -24,6 +24,7 @@
 exclude_patterns = ['_includes/*.rst']
 html_theme = 'sylius_rtd_theme'
 html_theme_path = ["_themes"]
+html_favicon = 'favicon.ico'
```

Comment:
Alternatively, it could be
`html_favicon = '../public/favicon.ico`
and a duplicated favicon.ico file wouldn't be needed, but I wasn't sure about docs deployment and out of docs directory scope